### PR TITLE
mw - adds hello world page content and corresponding tests

### PR DESCRIPTION
**Pull request by mattwebbio**
Closes #1

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [x] I have replaced `X` above with the associated issue number.
- [x] I have changed the issue's label to "pull request".
- [x] I have assigned the designated reviewer to this PR.
- [x] I have set the appropriate label on this PR.
- [x] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

File: app/assets/javascripts/home.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (darreneng) [inline app/assets/javascripts/home.js]: we probably don't need this

File: app/assets/stylesheets/home.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the home controller here.
```
Comment (darreneng) [inline app/assets/stylesheets/home.scss]: don't need this either

File: app/helpers/home_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```
Comment (darreneng) [inline app/helpers/home_helper.rb]: don't need this

**Comment by mattwebbio** [comment]
[Issue 1: Development Environment](https://trello.com/c/HzpdBZAC/214-issue-1-development-environment)

**Comment by mattwebbio** [comment]
[Issue 1: Development Environment](https://trello.com/c/HzpdBZAC/214-issue-1-development-environment)

**Pull request merged**